src/solve.jl

```diff
@@ -6,6 +6,17 @@ function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase
   integrator.sol
 end
 
+tspan2dtmin(tspan) = tspan2dtmin(tspan, one(eltype(tspan)))
```

Comment:
should this go to DiffEqBase and be uniformly applied to *DiffEq?

Reply:
https://github.com/JuliaDiffEq/DiffEqBase.jl/pull/438